diff --git a/demo/emulator-run.ts b/demo/emulator-run.ts
index 55fe8e7..a7f52ee 100644
--- a/demo/emulator-run.ts
+++ b/demo/emulator-run.ts
@@ -6,7 +6,7 @@ import { GDBTCPServer } from '../src/gdb/gdb-tcp-server.js';
 
 // Create an array with the compiled code of blink
 // Execute the instructions from this array, one by one.
-const hex = fs.readFileSync('hello_uart.hex', 'utf-8');
+const hex = fs.readFileSync('solarreader.hex', 'utf-8');
 const mcu = new RP2040();
 mcu.loadBootrom(bootromB1);
 loadHex(hex, mcu.flash, 0x10000000);
